backend/services/interfaces/testSessionService.ts

```diff
@@ -89,12 +89,12 @@ export interface ResultResponseDTO {
    * a list corresponding to the question list with each field indicating
    * the student's answer
    */
-  answers: (number | null)[];
+  answers: (number[] | number | null)[][];
   /**
    * a list corresponding to the question list with each fielding indicating
    * whether the student got the question right or not
    * */
-  breakdown: boolean[];
+  breakdown: boolean[][];
   /** the grading status of a result - either graded or ungraded (default) */
   gradingStatus: GradingStatus;
 }
```

Comment:
nit: should we update the function documentation to include explanations of updated functions? i.e `number[]` is used when there are sub-questions, `number` is used when there is one question, etc. since it's a bit non-obvious